tests/conftest.py

```diff
@@ -1776,18 +1782,30 @@ def label_registry(hass: HomeAssistant) -> lr.LabelRegistry:
 
 
 @pytest.fixture
-def service_calls() -> Generator[None, None, list[ServiceCall]]:
+def service_calls(hass: HomeAssistant) -> Generator[None, None, list[ServiceCall]]:
     """Track all service calls."""
     calls = []
 
+    _original_async_call = hass.services.async_call
+
     async def _async_call(
         self,
         domain: str,
         service: str,
         service_data: dict[str, Any] | None = None,
         **kwargs: Any,
-    ):
+    ) -> ServiceResponse:
         calls.append(ServiceCall(domain, service, service_data))
+        try:
+            return await _original_async_call(
+                domain,
+                service,
+                service_data,
+                **kwargs,
+            )
+        except ha.ServiceNotFound:
+            _LOGGER.debug("Ignoring unknown service call to %s.%s", domain, service)
+        return None
```

Comment:
Some tests rely on a real service call followed by a fake service call.

This ensures that the real service calls are logged and run, whilst fake service calls are only logged.